fitlins/interfaces/nistats.py

```diff
@@ -99,18 +125,20 @@ def _run_interface(self, runtime):
         fname_fmt = os.path.join(runtime.cwd, '{}_{}.nii.gz').format
         for name, weights, contrast_type in prepare_contrasts(
                 self.inputs.contrast_info, mat.columns.tolist()):
-            maps = flm.compute_contrast(weights, contrast_type, output_type='all')
             contrast_metadata.append(
                 {'contrast': name,
                  'stat': contrast_type,
                  **out_ents}
                 )
+            maps = flm.compute_contrast(
+                weights, contrast_type, output_type='all')
```

Comment:
Why?

Reply:
I think it just made sense to me to do that right before looping over it. No functional difference.